src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/mateacademy1
```

Comment:
1. Please separate groups of properties by a blank line
2. Why not use separate properties for a logger?
3. Does your logger work correctly?